pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -131,9 +134,20 @@ func (cm *containerManagerImpl) enforceNodeAllocatableCgroups() error {
 
 // enforceExistingCgroup updates the limits `rl` on existing cgroup `cName` using `cgroupManager` interface.
 func enforceExistingCgroup(cgroupManager CgroupManager, cName CgroupName, rl v1.ResourceList) error {
+	rp := getCgroupConfig(rl)
+
+	if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.MemoryQoS) {
+		if rp.Memory != nil {
+			if rp.Unified == nil {
+				rp.Unified = make(map[string]string)
+			}
+			rp.Unified[MemoryMin] = strconv.FormatInt(*rp.Memory, 10)
```

Comment:
This is to enforce the memory qos for node according to kube-reserved or system-reserved setttings, implementing points 1 & 2 at https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2570-memory-qos#node

Reply:
I see, it would be great to note this as a comment here or perhaps link to the KEP as you did in your comment.


One additional question, is this code path (i.e. `enforceExistingCgroup()`) called specifically only for when `enforce-node-allocatable=kube-reserved` and `enforce-node-allocatable=system-reserved` are set?